python/make_finemap_inputs.py

```diff
@@ -147,7 +147,7 @@ def read_sumstats(path,
 
 def filter_sumstat(df, bed):
     return pd.concat(
-        [df.loc[((df.chromosome == f.chrom) & (df.position >= f.start) & (df.position <= f.end)), :] for f in bed]
+        [df.loc[((df.chromosome.map(CHROM_MAPPING_INT).astype(int) == f.chrom) & (df.position >= f.start) & (df.position <= f.end)), :] for f in bed]
```

Comment:
The f.chrom needs to be transformed to int, as it's of type unicode here for whatever reason.
```suggestion
        [df.loc[((df.chromosome.map(CHROM_MAPPING_INT).astype(int) == int(f.chrom)) & (df.position >= f.start) & (df.position <= f.end)), :] for f in bed]
```

Reply:
good catch! fixed